firefox-ios/firefox-ios-tests/Tests/ClientTests/Microsurvey/MicrosurveyMiddlewareTests.swift

```diff
@@ -89,4 +102,41 @@ final class MicrosurveyMiddlewareTests: XCTestCase {
             actionType: actionType
         )
     }
+
+    private func setupAppState() -> AppState {
+        return AppState(
+            activeScreens: ActiveScreensState(
+                screens: [
+                    .browserViewController(
+                        BrowserViewControllerState(
+                            windowUUID: .XCTestDefaultUUID
+                        )
+                    ),
+                    .microsurvey(
+                        MicrosurveyState(
+                            windowUUID: .XCTestDefaultUUID
+                        )
+                    )
+                ]
+            )
+        )
+    }
+
+    private func setupTestingStore() {
```

Comment:
It looks good to me, i don't know only for later tests, if we want to add an Utility class to make a custom store and to restore it to default.

Reply:
thanks @FilippoZazzeroni! Yes, you're right, I agree. I created a separate ticket for this though since I prefer to break up work in smaller chunks: https://mozilla-hub.atlassian.net/browse/FXIOS-10094

I'll probably also get the teams input on the best way to go about this since want to make sure that its clear down the row that this is something we need to do for testing.

Reply:
Perfect thank you Cyndi, yes that is top.